homeassistant/components/elgato/light.py

```diff
@@ -59,10 +59,10 @@ def __init__(
         self.elgato = elgato
 
     @property
-    def name(self) -> str:
+    def name(self) -> str | None:
```

Comment:
```suggestion
    def name(self) -> str:
```

I don't think this can be `None`